### ISSUE TITLE
profile.lua does not exist

### ISSUE THREAD

**Issue by kandelb1** [kandelb1/itb-extrastats-mod#3]
It's possible for a profile to not have a `profile.lua`.
The vanilla stats browser is what reads the `profile.lua`. It should check if the file exists with `modApi:fileExists()` before attempting to read it with `modApi:loadIntoEnv()`.
If the file doesn't exist, just return an empty table in `fetchGameHistory()`.

**Comment by kandelb1** [comment]
This is the error reported by the discord user:
```./scripts/mod_loader/bootstrap/itb_io.lua:142: Failed to read file ".../Into The Breach/profile_Beta/profile.lua" as string: callback error
stack traceback:
    [C]: in function 'read_to_string'
    ./scripts/mod_loader/bootstrap/itb_io.lua:139: in function <./scripts/mod_loader/bootstrap/itb_io.lua:138>
    [C]: in function 'pcall'
    ./scripts/mod_loader/bootstrap/try_catch.lua:20: in function 'try'
    ./scripts/mod_loader/bootstrap/itb_io.lua:138: in function 'read_to_string'
    ./scripts/mod_loader/modapi/sandbox.lua:16: in function 'loadIntoEnv'
    ./mods/Extra Stats/scripts/vanilla_stats_browser.lua:18: in function 'fetchGameHistory'
    ./mods/Extra Stats/scripts/vanilla_stats_browser.lua:332: in function 'func'
    ./scripts/mod_loader/modui/modcontent.lua:41: in function 'onclicked'
    ./scripts/mod_loader/ui/widgets/base.lua:627: in function 'clicked'
    ./scripts/mod_loader/ui/widgets/root.lua:299: in function 'event'
    ./scripts/mod_loader/modui/root.lua:345: in function <./scripts/mod_loader/modui/root.lua:327>
caused by: File doesn't exist
```

**Comment by kandelb1** [comment]
Since we moved to reading/writing to the current profile's `modcontent.lua` instead of the `modcontent.lua` shared by all profiles (commit 15874a6), I think this issue is resolved. The modloader is responsible for creating that file anyway, so I don't think *Extra Stats* should be creating it if it doesn't exist.

**Comment by kandelb1** [comment]
I ran into this issue when testing, so I fixed it with commit e047c79.
